app/controllers/report_controller.rb

```diff
@@ -9,14 +9,17 @@ class ReportController < ApplicationController
   before_action :redirect_unless_moderator, except: [:create]
 
   def index
-    @reports = Report.where(reviewed: false)
+    @unreviewed_reports = Report.join_originator.where(reviewed: false).order(created_at: :desc)
+    @reviewed_reports = Report.join_originator.where(reviewed: true).order(created_at: :desc)
```

Comment:
I don't know if this is a good idea. Big pods can have a lot of reports, and loading all of them might be a performance problem, especially since this is always loaded, even if you only want to view new unreviewed reports.

So this either needs some sort of pagination, or at least only load the reviewed reports if you actually want to load that tab.

Reply:
Limiting for the last 100 for now.